### PR TITLE
Svelte + Vite example

### PR DESCRIPTION
**Pull request by RishhiB**
Simple Svelte Example, copied from the angular demo

### CODE REVIEW

File: svelte-demo/vite.config.ts
Comment (scottn12) [inline svelte-demo/vite.config.ts:1]: Could you add the MSFT copyright comment to the typescript/javascript files

File: svelte-demo/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import './app.css'
+import App from './App.svelte'
+
+const app = new App({
+  target: document.getElementById('app'),
+})
+
+export default app
```
Comment (scottn12) [inline svelte-demo/src/main.ts:8]:
```suggestion
import './app.css';
import App from './App.svelte';

const app = new App({
  target: document.getElementById('app'),
});

export default app;
```

File: svelte-demo/package.json
Comment (scottn12) [inline svelte-demo/package.json:1]: I think we want to integrate prettier into this demo?

**Review by scottn12 — COMMENTED** [review]
Looking good, thanks for starting this demo! One thing I didn't mention below is that we will probably want to add this demo to the `build.yml` file and potentially add some simple tests

**Comment by ChumpChief** [comment]
Looks like this PR is stale - if you're interested in pursuing this example further, let's discuss.  If so, it would at least need to be updated to FF 2.0 and we'd want to discuss what DDSs should be used (i.e. should it use SharedTree).

**Pull request closed without merging**